tests/python/driver/tvmc/test_command_line.py

```diff
@@ -289,3 +288,37 @@ def test_tvmc_print_pass_times(capsys, keras_simple, tmpdir_factory):
     captured_out = capsys.readouterr().out
     for exp_str in ("Compilation time breakdown by pass:", "sequential:", "us]"):
         assert exp_str in captured_out
+
+
+def test_tvmc_print_ir_before(capsys, keras_simple, tmpdir_factory):
+    pytest.importorskip("tensorflow")
+    tmpdir = tmpdir_factory.mktemp("out")
+    print_cmd = "--print-ir-before=[tir.SplitHostDevice]"
+
+    # Compile model
+    module_file = os.path.join(tmpdir, "keras-tvm.tar")
+    compile_cmd = f"tvmc compile --target 'llvm' {keras_simple} --output {module_file} {print_cmd}"
+    compile_args = compile_cmd.split(" ")[1:]
+    _main(compile_args)
+
+    # Check for timing results output
+    captured_out = capsys.readouterr().out
+    for exp_str in ("Print ir before:\n", "tir.SplitHostDevice\n"):
```

Comment:
It would be also good to check if something from the actual IR module is present, e.g. `prim_func`

Reply:
I checked @I.ir_module